docs/custom_promise_types/site_up.py

```diff
@@ -0,0 +1,45 @@
+import re
+import ssl
+import urllib.request
+import urllib.error
+from cfengine import PromiseModule, ValidationError, Result
+
+
+class SiteUpPromiseTypeModule(PromiseModule):
+    def __init__(self):
+        super().__init__("site_up_promise_module", "0.0.1")
+
+    def validate_promise(self, promiser, attributes):
+        if self.is_url_valid(promiser) == False :
```

Comment:
```suggestion
        if not self.is_url_valid(promiser):
```

More clean / pythonic.